board/batocera/rockchip/rk3326/dts/rk3326-odroid-go2-v11.dts

```diff
@@ -0,0 +1,701 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Copyright (c) 2019 Hardkernel Co., Ltd
+ * Copyright (c) 2020 Theobroma Systems Design und Consulting GmbH
+ */
+
+/dts-v1/;
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/pinctrl/rockchip.h>
+#include "rockchip/rk3326.dtsi"
+
+/ {
+	model = "ODROID-GO Advance";
+	compatible = "hardkernel,rk3326-odroid-go2", "rockchip,rk3326";
+
+	chosen {
+		stdout-path = "serial2:115200n8";
+	};
+
+	backlight: backlight {
+		compatible = "pwm-backlight";
+		power-supply = <&vcc_bl>;
+		pwms = <&pwm1 0 25000 0>;
+	};
+
+        battery_cell: battery-cell {
+                compatible = "simple-battery";
+                energy-full-design-microamp-hours = <3000000>;
+                voltage-min-design-microvolt = <3500000>;
+                voltage-max-design-microvolt = <4100000>;
+
+                ocv-capacity-celcius = <20>;
+                ocv-capacity-table-0 =  <3500 0>, <3625 5>, <3685 10>, <3697 15>,
+                                        <3718 20>, <3735 25>, <3748 30>, <3760 35>,
+                                        <3774 40>, <3788 45>, <3802 50>, <3816 55>,
+                                        <3834 60>, <3853 65>, <3877 70>, <3908 75>,
+                                        <3946 80>, <3975 85>, <4018 90>, <4071 95>,
+                                        <4106 100>;
+        };
+
+
+	utils {
```

Comment:
You might want to change these bindings to something that more closely resembles behavior before this PR.
What's in this dts is just what I wanted to have, with no particular thought given to anything